frontend/src/components/selects/CategorySelect.js

```diff
@@ -1,35 +1,81 @@
 import React from 'react';
-import { Button, MenuItem } from '@blueprintjs/core';
+import { Menu, Button, MenuItem } from '@blueprintjs/core';
 import { Select } from '@blueprintjs/select';
 import styled from 'styled-components';
 import PropTypes from 'prop-types';
+import deselectButton from '../../resources/deselect.png';
 
 const SelectContainer = styled.div``;
 
-const categories = ['Medical Equipment', 'World Vision', 'Bio Med', 'Kendall'];
+const ButtonWithBorder = styled.button`
+  display: block;
+  border-radius: 8px;
+  color: white;
+  background-color: white;
+  cursor: pointer;
+  margin-top: 2px;
+`;
 
-const itemRenderer = (category, { index, handleClick, modifiers }) => {
-  return (
-    <MenuItem
-      key={index + category}
-      onClick={handleClick}
-      text={category}
-      active={modifiers.active}
-      shouldDismissPopover={false}
-    />
-  );
-};
+const categories = ['Medical Equipment', 'World Vision', 'Bio Med', 'Kendall'];
 
 const CategorySelect = ({ onSelect, category }) => {
+  const deselectActive = () => {
+    onSelect('');
+  };
+  const SelectedItem = () => {
+    return (
+      <div>
+        <ButtonWithBorder type="submit" onClick={deselectActive}>
+          <img alt="deselect button" src={deselectButton} height="15" width="15" />
+        </ButtonWithBorder>
+      </div>
+    );
+  };
+
+  const itemListRenderer = ({ items, itemsParentRef, renderItem, activeItem }) => {
+    const renderedItems = items.map(renderItem).filter(item => item.props.text !== activeItem);
+    return (
+      <Menu ulRef={itemsParentRef}>
+        <MenuItem
```

Comment:
This MenuItem needs to be removed when activeItem is `''` bc all of the category items will be displayed in addition to this component. I think a cleaner way of handling this is to make a custom callback in `items.map` above checking if the mappited item == activeItem and calling renderItem if so and using the custom component otherwise.